_posts/0000-01-02-irrational-cat.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"I hope the only committing i'm doing in my life is committing to changes on github and not marriages or murders."